Packages/Timeline/Sources/Timeline/TimelineViewModel.swift

```diff
@@ -152,6 +154,15 @@ extension TimelineViewModel: StatusesFetcher {
                                                                   maxId: nil,
                                                                   minId: nil,
                                                                   offset: statuses.count))
+      if let account {
```

Comment:
I'm ok with the logic, should be refactored into a proper function like processStatuses or something, later we'll probably want to add more local stuff.

Reply:
Totally agree! On my list before remove the draft status.